### DISCUSSION TITLE
Help with encoder

### DISCUSSION THREAD

**Discussion by Anstouk** [tttapa/Control-Surface#967] [Q&A]
using AbsoluteEncoder when the encoder is turned to the minimum value, the value stops at 0. How to make the value 0 continue to be transmitted when turned to minimum, and when the maximum value is reached. 
example  2,1,0,0,0...

**Comment by tttapa** [comment]
You'll have to remove this check:

https://github.com/tttapa/Control-Surface/blob/ba8812f571df1ee4480eb20b9c5c4edabebfc72d/src/MIDI_Outputs/Abstract/MIDIAbsoluteEncoder.hpp#L41
